### PR TITLE
VGC_2S class modification

### PR DESCRIPTION
**Pull request by Mbosum**
## Description

Modified VGC_2S class to inherit from VRC and allow for user to change at_vac setpoint value for upstream and downstream gauges separately

## Motivation and Context

Change the UI to at_vacuum_setpoint_us  and at_vacuum_setpoint_ds  to reduce confusion for users.

## How Has This Been Tested?

```
(pcds-5.0.0) tmo-control:pcdsdevices bosum123$ PYTHONPATH=/reg/g/pcds/epics-dev/bosum123/hutch_python/pcdsdevices:$PYTHONPATH ipython
Python 3.9.7 | packaged by conda-forge | (default, Sep 29 2021, 19:20:46)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.28.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from pcdsdevices.valve import VGC_2S

In [2]: exit()
```
```
(pcds-5.0.0) tmo-control:pcdsdevices bosum123$ flake8 valve.py
(pcds-5.0.0) tmo-control:pcdsdevices bosum123$
```

## Where Has This Been Documented?

There is a docstring for the class.

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [ ] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

File: pcdsdevices/valve.py [hunk 1]
```diff
@@ -154,16 +154,44 @@ class VGC(VRC):
...
+    ext_ilk_ok = Cpt(EpicsSignalRO, ':EXT_ILK_OK_RBV', kind='normal',
+                     doc='External interlock ok')
+
+    at_vac_setpoint_us = Cpt(EpicsSignalWithRBV, ':AT_VAC_SP_US',
+                             kind='config',
+                             doc='AT VAC Set point value '
+                             'for the upstream gauge')
+    setpoint_hysterisis_us = Cpt(EpicsSignalWithRBV, ':AT_VAC_HYS_US',
+                                 kind='config',
+                                 doc='AT VAC Hysterisis for '
```
Comment (klauer) [inline pcdsdevices/valve.py:170]:
```suggestion
                                 doc='AT VAC Hysteresis for '
```

File: pcdsdevices/valve.py [hunk 2]
```diff
@@ -154,16 +154,44 @@ class VGC(VRC):
...
+                                 'the upstream setpoint')
     at_vac_setpoint_ds = Cpt(EpicsSignalWithRBV, ':AT_VAC_SP_DS',
                              kind='config',
                              doc='AT VAC Set point value '
-                                 'for the downstream gauge')
+                             'for the downstream gauge')
     setpoint_hysterisis_ds = Cpt(EpicsSignalWithRBV, ':AT_VAC_HYS_DS',
                                  kind='config',
                                  doc='AT VAC Hysterisis for '
```
Comment (klauer) [inline pcdsdevices/valve.py:178]:
```suggestion
                                 doc='AT VAC Hysteresis for '
```

File: pcdsdevices/valve.py [hunk 3]
```diff
@@ -154,16 +154,44 @@ class VGC(VRC):
...
-                                     'the downstream setpoint')
+                                 'the downstream setpoint')
+
+    at_vac = Cpt(EpicsSignalRO, ':AT_VAC_RBV', kind='normal',
+                 doc='at vacuum sp is reached')
```
Comment (klauer) [inline pcdsdevices/valve.py:182]:
```suggestion
                 doc='at vacuum setpoint is reached')
```

File: pcdsdevices/valve.py [hunk 4]
```diff
@@ -154,16 +154,44 @@ class VGC(VRC):
...
+    error = Cpt(EpicsSignalRO, ':ERROR_RBV', kind='normal',
+                doc='Error Present')
+    mps_state = Cpt(EpicsSignalRO, ':MPS_FAULT_OK_RBV', kind='omitted',
+                    doc=('individual valve MPS state for debugging'))
+    interlock_device_upstream = Cpt(EpicsSignalRO, ':ILK_DEVICE_US_RBV',
```
Comment (klauer) [inline pcdsdevices/valve.py:187]: We unfortunately don't provide enough examples of using the following coding style (that is to say it's probably our fault as the PD group), but consider how much more readable this is:
```python
    interlock_device_upstream = Cpt(
        EpicsSignalRO,
        ":ILK_DEVICE_US_RBV",
        kind="config",
        string=True,
        doc="Upstream vacuum device used for interlocking this valve",
    )
```

That said, you don't have to fix these up - but maybe keep this in mind for the future.

File: docs/source/upcoming_release_notes/884-vgc_2s.rst [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- Bosum123
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/884-vgc_2s.rst:30]: github username
```suggestion
- MBosum
```

**Review by ZLLentz — APPROVED** [review]
The diff is actually really difficult to follow now since you expanded your scope from "modified one class" to "let's fix the formatting everywhere". I think we should avoid bundling in mass reformatting with a functional PR. I think everything here is ready to merge, except the pre-release notes have no context. I'm going to merge anyway and fix it myself when we go to make the tag since I want to clear the pcdsdevices PR chain.

**Pull request merged**